kahuna/public/js/search/index.js

```diff
@@ -47,11 +52,16 @@ search.config(['$stateProvider',
                 });
                 return {state: $dsr$.redirect.state, params};
             }]
+        },
+        resolve: {
+            searchQueryService: ['$stateParams', 'searchQueryService', function($stateParams, searchQueryService) {
```

Comment:
Yea, I was trying not to break things while refactoring, I'll get on the naming...